SharedSource/Development/UnitTesting/ObjectExtensions.cs

```diff
@@ -35,25 +35,25 @@ public static T As<T> (this object obj)
     }
 
     [DebuggerStepThrough]
-    public static T Invoke<T> (this object target, string method, params object[] args)
+    public static T? Invoke<T> (this object target, string method, params object[] args)
     {
-      return (T) PrivateInvoke.InvokeNonPublicMethod (target, method, args);
+      return (T?) PrivateInvoke.InvokeNonPublicMethod (target, method, args);
     }
 
     [DebuggerStepThrough]
-    public static object Invoke (this object target, string method, params object[] args)
+    public static object? Invoke (this object target, string method, params object[] args)
     {
       return PrivateInvoke.InvokeNonPublicMethod (target, method, args);
```

Comment:
given the descripton on MethodInfo.Invoke about args being nullable, we probably want to update InvokeNonPublicMethod and the signatures here, too.